uls-fetch.sh

```diff
@@ -133,6 +133,7 @@ main() {
 
 		al_type="${fn#*_}"
 		al_type="${al_type%%_*}"
+		al_type="${al_type%%.info}"
```

Comment:
Nice catch on the suffix removal.

Could this be changed to either shortest-trailing, `%.info` or longest-trailing `%%.*` instead? A non-glob string won't benefit from longest-match. Between the above options, a glob makes it easier for alternative suffixes to be used in the future (like `.info.txt`) but may impair meaning as this is already hard-coded elsewhere. Either works for me as more direct, though.